.snyk

```diff
@@ -0,0 +1,21 @@
+# Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
+version: v1.13.5
+# ignores vulnerabilities until expiry date; change duration by modifying expiry date
+ignore:
+  SNYK-JS-LODASH-450202:
+    - '@loopback/boot > @loopback/repository > loopback-datasource-juggler > lodash':
+        reason: None given
```

Comment:
@JamilOmar - For future reference we need to give reason why we ignore the module. Copy the description from snyk report and paste here.

Reply:
ok